frameworks/move-stdlib/sources/vector.move

```diff
@@ -604,6 +610,36 @@ module std::vector {
         for_each_reverse(v, |e| d(e))
     }
 
+    public fun range(start: u64, end: u64): vector<u64> {
+        range_with_step(start, end, 1)
+    }
+
+    public fun range_with_step(start: u64, end: u64, step: u64): vector<u64> {
```

Comment:
Need to check the size relationship between start, end and step parameters.

Reply:
There is an assert `step > 0`. If end > start, the result is empty.